### PR TITLE
[feature]Internationalization of transformation management console

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

Internationalization of management console parameter configuration

### Related issues/PRs

Related issues: #3875
Related pr:#3825

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/CategoryService.scala [hunk 1]
```diff
@@ -107,14 +107,17 @@ class CategoryService extends Logging {
     firstCategoryList
   }
 
-  def getAllCategory(): util.List[CategoryLabelVo] = {
+  def getAllCategory(language: String): util.List[CategoryLabelVo] = {
     val categoryLabelList = configMapper.getCategory()
     val categoryLabelTreeList = buildCategoryTree(categoryLabelList)
+    if (StringUtils.isEmpty(language) || (!language.equals("en"))) {
```
Comment (casionone) [inline linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/CategoryService.scala:113]:
```
if (StringUtils.isEmpty(language) || (!language.equals("en"))
->
 if ("en".equals(language))
```
？

File: linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/ConfigurationService.scala [hunk 2]
```diff
@@ -316,17 +316,28 @@ class ConfigurationService extends Logging {
...
+      language: String
   ): (util.List[ConfigKeyValue], util.List[ConfigKeyValue]) = {
     LabelParameterParser.labelCheck(labelList)
     val combinedLabel = combinedLabelBuilder.build("", labelList).asInstanceOf[CombinedLabelImpl]
     val label =
       labelMapper.getLabelByKeyValue(combinedLabel.getLabelKey, combinedLabel.getStringValue)
     var configs: util.List[ConfigKeyValue] = new util.ArrayList[ConfigKeyValue]()
     if (label != null && label.getId > 0) {
-      configs = configMapper.getConfigKeyValueByLabelId(label.getId)
+      configs = getConfigByLabelId(language, label.getId)
```
Comment (casionone) [inline linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/ConfigurationService.scala:340]: The key parameters are recommended to be placed in front 
with default vaule?

 def getConfigByLabelId(labelId: Integer, language: String ="zh-CN"): util.List[ConfigKeyValue] = {
    var configs: util.List[ConfigKeyValue] = new util.ArrayList[ConfigKeyValue]()
    if ("en".equals(language)) {
      configs = configMapper.getConfigEnKeyValueByLabelId(labelId)
    } else {
      configs = configMapper.getConfigKeyValueByLabelId(labelId)
    }
    configs
  }
```

File: linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/dao/impl/ConfigMapper.xml [hunk 3]
```diff
@@ -92,6 +92,11 @@
         v.id as value_id, v.config_key_id,config_value, v.config_label_id
     </sql>
 
+    <sql id="Config_En_Key_Value_Column">
+        k.id, k.key, k.en_description as description, k.en_name as name, k.engine_conn_type, k.default_value, k.validate_type, k.validate_range, k.is_hidden, k.is_advanced, k.level, k.en_treeName as treeName,
+        v.id as value_id, v.config_key_id,config_value, v.config_label_id
```
Comment (casionone) [inline linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/dao/impl/ConfigMapper.xml:97]: Try to control no more than three fields per line

File: linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/CategoryService.scala (hunk 1 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/CategoryService.scala:113]: done

File: linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/ConfigurationService.scala (hunk 2 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-configuration/src/main/scala/org/apache/linkis/configuration/service/ConfigurationService.scala:340]: done

File: linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/dao/impl/ConfigMapper.xml (hunk 3 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/dao/impl/ConfigMapper.xml:100]: done

**Comment by casionone** [comment]
code format

**Review by casionone — CHANGES_REQUESTED** [review]
the change of ddl and dml  need to record

**Pull request closed without merging**